addons/web/static/src/js/views/graph/graph_model.js

```diff
@@ -35,7 +35,7 @@ return AbstractModel.extend({
      * @returns {Object}
```

Comment:
split into two commits: one for the sample data and one with the replacement of nocontent helpers by empty charts

Reply:
It was done mainly in order to avoid inconsistency in dashboard